src/components/screensavermanager.js

```diff
@@ -20,27 +20,26 @@ define(["events", "playbackManager", "pluginManager", "inputManager", "connectio
         }
     });
 
-    function getScreensaverPlugin(isLoggedIn) {
+    function getScreensaverPlugin() {
 
         var option;
-        var defaultOption = isLoggedIn ? "backdropscreensaver" : "logoscreensaver";
         try {
             option = userSettings.get("screensaver", false);
         } catch (err) {
             option = null;
```

Comment:
maybe we should define it here using info if user is logged in?..

Reply:
why?

Reply:
in line 89 you're calling it like this:
```js
var screensaver = getScreensaverPlugin(isLoggedIn);
```
but your latest implementation of `getScreensaverPlugin()` doesn't take any parameters

Reply:
https://github.com/jellyfin/jellyfin-web/pull/592/commits/0821d0ad9edffa9570723750a547f0cb764df73f